Formula/s/screenpipe.rb

```diff
@@ -0,0 +1,58 @@
+class Screenpipe < Formula
+  desc "Library to build personalized AI powered by what you've seen, said, or heard"
+  homepage "https://github.com/mediar-ai/screenpipe"
+  url "https://github.com/mediar-ai/screenpipe/archive/refs/tags/v0.1.98.tar.gz"
+  sha256 "cb3c8039ecb60d35bacd2b9673db112f907b4a1d3d7c32f49a5e77c0274268ad"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "dbus"
+  depends_on "ffmpeg"
+  depends_on "libxcb"
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "tesseract"
+  end
+
+  def install
+    features = ["--features", "metal,pipes"] if OS.mac? && Hardware::CPU.arm?
+    system "cargo", "install", *features, *std_cargo_args(path: "screenpipe-server")
+    lib.install "screenpipe-vision/lib/libscreenpipe_#{Hardware::CPU.arch}.dylib" if OS.mac?
```

Comment:
Is there a Linux library too?

Reply:
removed the if OS.mac? check as it was unnecessary because the library works on both.

Reply:
It always names the extension `.dylib` even if the OS doesn't use `.dylib` files?

Reply:
That's very curious. Is there any reason from upstream why they don't build a library on Linux?

Reply:
`libscreenpipe_x86_64.dylib` is apple OCR code which is only for apple, not for linux

linux uses Tesseract OCR engine